test/mix/tasks/exfmt_test.exs

```diff
@@ -4,33 +4,43 @@ defmodule Mix.Tasks.ExfmtTest do
 
   describe "mix exfmt" do
     test "with no args" do
-      io = capture_io(fn->
+      io = capture_io(:stderr, fn->
```

Comment:
Seems we flip-flop between testing the mix task and testing the pure functional abstraction. Could we test only the abstraction? :)